package.json

```diff
@@ -62,7 +62,8 @@
     ],
     "collectCoverage": true,
     "coveragePathIgnorePatterns": [
-      "dist/"
+      "dist/",
+      "src/helpers/webp-support.js"
```

Comment:
webp-support.js file is ignored for coverage, because WebP detect uses a canvas element to detect support and jsdom in Jest test cycle doesn’t support canvas DOM element.